### PR TITLE
fixes #4635 fix(Legacy): Add default for total for normandy sampling …

### LINKED ISSUE #4635: [Legacy] Experimenter does not handle normandy namespace samples
Namespace samples do not include a total in its filter object, while experimenter assumes it exists

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-963)
┆Issue Number: EXP-963

### PR DESCRIPTION
**Pull request by tiftran**
…filter

Because:

* the namespace filter does not include a total value

This commit:

* add default value of 10000 for total for sampling filters

### CODE REVIEW

File: app/experimenter/normandy/tasks.py [hunk 1]
```diff
@@ -192,9 +192,10 @@ def update_population_percent(experiment, recipe_data, filter_objects):
     ):
 
         decimal.getcontext().prec = 5
+        bucket_total = bucket_sample.get("total", 10000)
```
Comment (jaredlockhart) [inline app/experimenter/normandy/tasks.py:195]: Move this into `constants/legacy.py`?

**Review by jaredlockhart — APPROVED** [review]
👍

File: app/experimenter/normandy/tasks.py (hunk 1 above)
Comment (tiftran) [reply app/experimenter/normandy/tasks.py:195]: hah, it already exists!

Comment (jaredlockhart) [reply app/experimenter/normandy/tasks.py]: ;)

**Pull request merged**